### PR TITLE
add fork test file

### PR DESCRIPTION
**Pull request by corgiboy-g**
add new fork test file

### CODE REVIEW

**Comment by corgiboygsj** [comment]
add some other code

**Review by corgiboygsj — CHANGES_REQUESTED** [review]
add new method name test

File: src/forkTest/ForkTest.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+/**
+ * @author guoshoujing
+ * @create 2021-03-08 上午10:49
+ */
+public class ForkTest {
+    public static void main(String[] args) {
+        System.out.println("I'm a fork test file");
+        test();
+    }
```
Comment (corgiboygsj) [inline src/forkTest/ForkTest.java:11]: 添加注释

File: src/forkTest/ForkTest.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    public static void test(){
+        System.out.println("this method is test");
+    }
```
Comment (corgiboygsj) [inline src/forkTest/ForkTest.java:15]: 加一行代码

Comment (corgiboy-g) [reply src/forkTest/ForkTest.java:15]: ok

File: src/forkTest/ForkTest.java (hunk 1 above)
Comment (corgiboygsj) [reply src/forkTest/ForkTest.java:11]: > 添加注释

123

**Pull request merged**